pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +337,234 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+
+class DualDenseRKPseudoStepper(BaseDualPseudoStepper):
+    @property
+    def _pseudo_stepper_has_lerrest(self):
+        return False
+
+    def step(self, t):
+        add = self._add
+        rhs = self._rhs_with_dts
+        dtau = self._dtau
+
+        r = self._pseudo_stepper_regidx
+
+        # Ensure r0 references the bank containing u(n+1,m)
+        if r[0] != self._idxcurr:
+            r[0], r[1] = r[1], r[0]
+
+        # First stage
+        rhs(t, r[0], r[1])
+
+        # Other stages
+        for i in range(self._pseudo_stepper_nregs - 2):
+            ar = [ar for ars in zip([a*dtau for a in self.a[i]],
```

Comment:
This could still be simplifiedl

Reply:
Do you have any suggestions? I can't think of a nice alternative one liner without using numpy. Thanks.

Reply:
chain from itertools is what you need.  Specifically, `ar = chain(*zip(..., ...))`.  May want to factor out the multiplications by dtau to the top of the function.  So `a = [self._dtau*ai for ai in self.a]`.  Also saves some needless multiplications in the inner for loop.